Source/init.cpp

```diff
@@ -199,6 +212,13 @@ void init_archives()
 	}
 }
 
+void init_language_archives()
+{
+	lang_mpq = std::nullopt;
+	auto paths = GetMPQSearchPaths();
+	init_language_archives(paths);

```

Comment:
Once the argument has been made const, we can simply do this thanks to reference lifetime extension.

```suggestion
	init_language_archives(GetMPQSearchPaths());
```

Reply:
Fixed

Reply:
wasent there a bug in windows regarding this?